Taskfile.yml

```diff
@@ -66,6 +66,21 @@ tasks:
       - task: generate-rlp
       - task: check-clean-branch
 
+  check-generate-bindings:
+    desc: Checks that generated Go bindings from Solidity contracts in precompile are up-to-date (requires a clean git working tree)
+    cmds:
+      - cmd: |
+          # Find and remove generated binding files in precompile directories only
+          find ./precompile -type f -name 'gen_*binding.go' -exec grep -l '^// Code generated - DO NOT EDIT\.$' {} \; | xargs -r rm
```

Comment:
why are we removing the file? isn't it sufficient to just regenerate it then check check clean branches like in generate-codec